integrations/access/servicenow/app.go

```diff
@@ -75,13 +70,8 @@ type App struct {
 
 // NewServicenowApp initializes a new teleport-servicenow app and returns it.
 func NewServiceNowApp(ctx context.Context, conf *Config) (*App, error) {
-	teleportClient, err := conf.GetTeleportClient(ctx)
```

Comment:
Someone's welcome to explain to me why it's necessary for the Teleport client to be created both in `NewServiceNowApp` and `(a *App) init` - I stared at this for about half an hour before deciding I'd just delete it from this point and see if things still work.